rusty_pizza_server/src/order_model/user.rs

```diff
@@ -1,11 +1,18 @@
+use crate::util::id::Id;
+
 #[derive(Debug, PartialEq, Eq, Hash)]
 pub struct User {
+    id: Id,
     name: String,
 }
 
 impl User {
-    pub fn new(name: String) -> User {
-        User { name }
+    pub fn new(id: Id, name: String) -> User {
+        User { id, name }
+    }
```

Comment:
The id of an User should be genarated at some point.

Reply:
I agree, but imo this is out of scope of this PR. Most comfortable would be having some kind of "single sign on" with, e.g. an LDAP account or similar. In that case we need to somehow map between our user IDs and the external user identification data. I'd like to tackle sign in and the associated user management at a later stage, leaving this open for now. We can open an issue, but I think it is too vague what to do to specify it yet.